editor/editor_log.cpp

```diff
@@ -316,6 +346,66 @@ void EditorLog::_reset_message_counts() {
 	}
 }
 
+void EditorLog::_resource_path_clicked(Variant path) {
+	if (path.get_type() == Variant::Type::STRING) {
+		String path_str = static_cast<String>(path);
+
+		PackedStringArray split_by_colon = path_str.split(":");
+		String path_base = split_by_colon[1]; // Used to get the path without line numbers, if the path so happens to have them.
+		String file_type = EditorFileSystem::get_singleton()->get_file_type("res:" + path_base);
```

Comment:
There are lots of `"res:" + path_base` in this function. I think it's better to make it a variable.

Reply:
Fixed!